src/ResourceManager/Sql/AzureRM.Sql.psd1

```diff
@@ -9,7 +9,7 @@
 @{
```

Comment:
Please update the markdown help using: https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md

Please add a section to the changelog here: https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Sql/ChangeLog.md